roles/kubernetes-apps/external_cloud_controller/openstack/templates/external-openstack-cloud-config.j2

```diff
@@ -36,6 +36,12 @@ subnet-id={{ external_openstack_lbaas_subnet_id }}
 {% if external_openstack_lbaas_floating_network_id is defined %}
 floating-network-id={{ external_openstack_lbaas_floating_network_id }}
 {% endif %}
-{% if external_openstack_lbaas_flaoting_subnet_id is defined %}
+{% if external_openstack_lbaas_floating_subnet_id is defined %}
 floating-subnet-id={{ external_openstack_lbaas_floating_subnet_id }}
 {% endif %}
+{% if external_openstack_lbaas_manage_security_groups is defined %}
+manage-security-groups={{ external_openstack_lbaas_manage_security_groups }}
```

Comment:
I didn't include it because it is not needed when using Octavia, which is the default for LBaaS because the Neutron LBaaSv2 is deprecated.
Do you have a workload where you are using LBaaSv2?
And if it is the case, you would need to also specify the node-security-group as specified in: https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/provider-configuration.md#load-balancer-optional-parameters

Reply:
Yeah I'm using LBaaSv2 because Octavia is not yet supported on my platform.

`node-security-group` is not used anymore in the code. The Openstack docs are out-of-date unfortunately. Luckily the security group functionality works fine without it, as the CCM can automatically resolve which security group a node is in.

Reply:
I see, thanks for the precision

Reply:
Thank you for the review!